recipes/conch_sounds/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "conch_sounds" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/conch-sounds-{{ version }}.tar.gz
+  sha256: 8dffbbf154baec8f265eaa0205b26f6edfcf0a39f94bb588cc540c8aa244fe30
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - librosa >=0.8.0
+    - pip
+    - pyraat
+    - pytest
+    - python >=3.6
+  run:
+    - librosa >=0.8.0
+    - numpy
+    - praatio >=5.0
```

Comment:
Upstream uses `~=` do you know which one is correct? (Note: conda-build doesn't support `~=` so that would need to be translated into the respective `>=,=` version).

Reply:
">=,=" should be correct. I'll test.